charts/caraml-store/values.yaml

```diff
@@ -0,0 +1,169 @@
+# Default values for caraml-store.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+registry:
+  enabled: true
```

Comment:
this is never used

Reply:
Updated the templates to check for the flag.